src/components/application_manager/test/commands/mobile/subscribe_vehicle_data_request_test.cc

```diff
@@ -117,6 +117,19 @@ MATCHER_P6(CheckResponseWithKeyParams,
          key_result_code == key_code;
 }
 
+class TestSubscribeVehicleDataRequest : public SubscribeVehicleDataRequest {
+ public:
+  TestSubscribeVehicleDataRequest(const MessageSharedPtr& message,
+                                  am::ApplicationManager& application_manager)
+      : SubscribeVehicleDataRequest(message, application_manager) {}
+
+  virtual ~TestSubscribeVehicleDataRequest() {}
```

Comment:
@LevchenkoS why do you need empty virtual destructor? Does SubscribeVehicleDataRequest have not destructor?